content/file/276ba89e-9679-45df-b6d2-0a6b81b71105.json

```diff
@@ -1,7 +1,7 @@
 {
     "_links": {
         "self": {
-            "href": ""
+            "href": "http:\/\/default\/sites\/default\/files\/media\/image\/photo-epel-stanford-campus.jpg"
```

Comment:
Whats up with this path?

Reply:
hmm. no idea. something with serializer :man_shrugging: